working-groups/wg-run-discourse-on-tidb.md

```diff
@@ -0,0 +1,29 @@
+# Run Discourse on TiDB Working Group
+
+## Goals
+
+Run Discourse on TiDB instead of PostgreSQL
+
+## Meetings
+
+* Regular Working Group Meeting: [UTC+8]2:30pm on every Thursday
+* [Meeting Notes](https://docs.google.com/document/d/1tQPOSb4kTVvA-mAouFr9gdDSPotqJNzgM--caNBZQHU/edit)
+
+## Mentors
+
+* [sykp241095](http://github.com/sykp241095), [PingCAP](https://github.com/pingcap)
+* [hooopo](http://github.com/hooopo), member of organizers in Ruby-China's Community
+
+## Organizers
+
+* [sykp241095](http://github.com/sykp241095), [PingCAP](https://github.com/pingcap)
+
+## Members
+
+* [wd0517](https://github.com/wd0517), [PingCAP](https://github.com/pingcap)
+* [JinLingChristopher](https://github.com/JinLingChristopher)
+
+## Contact
+
+* Slack: channel **#wg-run-discourse-on-tidb** in the
+  [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```

Comment:
slack url: https://pingcap.com/tidbslack/

Reply:
this is not an invitation link